.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,3 +13,5 @@ Resolves #
 - [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
 - [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
 - [ ] `make vendor` does not cause changes.
+- [ ] Godoc-style comments have been added to (1) new code additions, and (2) when existing code
```

Comment:
Is there a `godoc` command we could recommend running to see what comments look like? If it's more than a single line, maybe we could add a section on go docs in the [dev guide](/Documentation/development-flow.md)?